rero_ils/modules/patrons/views.py

```diff
@@ -153,7 +153,10 @@ def logged_user():
 @register_menu(
     blueprint,
     'main.profile.patron_profile',
-    _('%(icon)s Profile', icon='<i class="fa fa-book fa-fw"></i>'),
+    text='{icon} {profile}'.format(
+            icon='<i class="fa fa-book fa-fw"></i>',
+            profile=_('Profile')
```

Comment:
Are you sure about this option ? I tried myself a solution looks like this one 2 days ago and it fails. It seems due to the `@register_menu` decorator executed once at the beginning of the app. So as the current language are not yet defined, the default "Profile" string is used and doesn't change in the menu if I changed the language.

Reply:
No I'm not sure. I did it this way because it's what I found in rero_ils/views, but as 'Profile' is not translated in the app, I can't check that it's ok. I was about to ask something about that during the daily, that's why I didn't ask for review for the moment.